src/main/java/com/intuit/graphql/authorization/rules/RuleParser.java

```diff
@@ -6,5 +6,5 @@
 
 public interface RuleParser {
 
-  Map<String, Set<String>> parseRule(String rule);
+  Map<String, Set<String>> parseRule(String rule, InvalidFieldsCollector invalidFieldsCollector);
```

Comment:
Shouldnt this be RuleParserListener?

Reply:
👌

Reply:
I changed the interface to `parseRule(String rule,  RuleParserListener ruleParserListener)`